File: C, C++/3224_Aaah.c
#include <stdio.h>
#include <string.h>

int main(){
    char a[1000],b[1000];
    scanf("%s%s", &a,&b);
    if( strlen(a) >= strlen(b)){
        printf("go\n");
    }
    else{
        printf("no\n");
    }
}

File: golang/3224_Aaah.go
package main

import "fmt"

func main() {
	var a, b string
	fmt.Scanf("%s\n%s", &a, &b)
	if len(a) >= len(b) {
		fmt.Println("go")
	} else {
		fmt.Println("no")
	}

}
